GeeksCoreLibrary/Core/Services/WiserItemsService.cs

```diff
@@ -2424,14 +2424,20 @@ public async Task<List<WiserItemModel>> GetLinkedItemDetailsAsync(IWiserItemsSer
                 where.Add("(permission.id IS NULL OR (permission.permissions & 1) > 0)");
             }
 
+            var tablePrefixItemId = "";
+            if (linkSettings.UseItemParentId && reverse && !string.IsNullOrWhiteSpace(itemIdEntityType))
+            {
+                tablePrefixItemId = await wiserItemsService.GetTablePrefixForEntityAsync(itemIdEntityType);
+            }
+            
             var tablePrefix = "";
-            if (!String.IsNullOrWhiteSpace(entityType))
+            if (!string.IsNullOrWhiteSpace(entityType))
```

Comment:
please use String.IsNullOrWhiteSpace instead of string.IsNullOrWhiteSpace

Reply:
@MikeVanMourik , you mean string.IsNullOrEmpty instead of IsNullOrWhiteSpace. I have not changed anything in this functionality, it was already this way.

Reply:
yes sorry i meant "IsNullOrWhiteSpace" just make sure its String and not string